src/ccextractor.c

```diff
@@ -41,8 +41,8 @@ void print_end_msg(void)
 
 int api_start(struct ccx_s_options api_options)
 {
-    struct lib_ccx_ctx *ctx;
-    struct lib_cc_decode *dec_ctx = NULL;
+    struct lib_ccx_ctx *ctx;              // Context for libs
```

Comment:
Probably a good idea to initialize it with NULL.

Reply:
Done.